account_invoice_partner_no_autofollow/tests/test_account_move_no_autofollow.py

```diff
@@ -0,0 +1,111 @@
+            "product_uom_id": cls.product_b.uom_id.id,
+            "quantity": 1.0,
+            "discount": 0.0,
+            "price_unit": 160.0,
+            "price_subtotal": 160.0,
+            "price_total": 208.0,
+            "tax_ids": cls.product_b.supplier_taxes_id.ids,
+            "tax_line_id": False,
+            "currency_id": False,
+            "amount_currency": 0.0,
+            "debit": 160.0,
+            "credit": 0.0,
+            "date_maturity": False,
+            "tax_exigible": True,
+        }
+
+    def test_in_invoice_create(self):
+        self.env["ir.config_parameter"].sudo().set_param("account.no_auto_follow", True)
+        move = self.env["account.move"].create(
+            {
+                "type": "in_invoice",
+                "partner_id": self.partner_a.id,
+                "invoice_date": fields.Date.from_string("2022-11-04"),
+                "currency_id": self.currency_data["currency"].id,
+                "invoice_payment_term_id": self.pay_terms_a.id,
+                "invoice_line_ids": [
+                    (0, None, self.product_line_vals_1),
+                    (0, None, self.product_line_vals_2),
+                ],
+            }
+        )
+
+        self.assertEqual(
+            len(move.message_follower_ids.ids), 1, msg="Count followers must be equal 0"
+        )
+
+    def test_in_invoice_write(self):
+        self.env["ir.config_parameter"].sudo().set_param("account.no_auto_follow", True)
+
+        move = self.env["account.move"].create(
+            {
+                "type": "in_invoice",
+                "partner_id": self.partner_a.id,
+                "invoice_date": fields.Date.from_string("2022-11-04"),
+                "currency_id": self.currency_data["currency"].id,
+                "invoice_payment_term_id": self.pay_terms_a.id,
+                "invoice_line_ids": [(0, None, self.product_line_vals_1)],
+            }
+        )
+        move.write({"invoice_line_ids": [(0, None, self.product_line_vals_2)]})
+
+        self.assertEqual(
+            len(move.message_follower_ids.ids), 1, msg="Count followers must be equal 0"
+        )
+
+    def test_in_invoice_action_post(self):
+        self.env["ir.config_parameter"].sudo().set_param("account.no_auto_follow", True)
+        move = self.env["account.move"].create(
+            {
+                "type": "in_invoice",
+                "partner_id": self.partner_a.id,
+                "invoice_date": fields.Date.from_string("2022-11-04"),
+                "currency_id": self.currency_data["currency"].id,
+                "invoice_payment_term_id": self.pay_terms_a.id,
+                "invoice_line_ids": [(0, None, self.product_line_vals_1)],
+            }
+        )
+        move.write({"invoice_line_ids": [(0, None, self.product_line_vals_2)]})
+        move.action_post()
+        self.assertEqual(
+            len(move.message_follower_ids.ids), 1, msg="Count followers must be equal 0"
```

Comment:
Why 1? Using fixed numbers here might play a bad trick on you.
What if some other module will add another partners as followers? Eg "accountant assistant" etc?